vuu-ui/showcase/src/examples/Popups/Notifications/Notifications.examples.tsx

```diff
@@ -0,0 +1,116 @@
+import { ChangeEvent, useState } from "react";
+import { Dropdown } from "@finos/vuu-ui-controls";
+import { NotificationsProvider, NotificationLevel, ToastNotification, useNotifications } from "@finos/vuu-popups";
+import { Input } from "@salt-ds/core";
+
+let displaySequence = 1;
+
+
+const Notifications = () => {
+    const [type, setType] = useState<NotificationLevel>(NotificationLevel.Info)
+    const [header, setHeader] = useState<string>("Header")
+    const [body, setBody] = useState<string>("Body")
+
+    const { notify } = useNotifications();
+
+    const handleNotification = () => {
+        notify({
+            type,
+            header,
+            body
+        })
+    }
+
+    return (
+        <div style={{ maxWidth: 500, display: 'flex', flexDirection: 'column' }}>
```

Comment:
Would this be better as a form or something?

Reply:
We could use salt's FormField/FormFieldLabel like we did for SaveLayoutPanel